.github/workflows/acceptance-tests-runner.yml

```diff
@@ -187,6 +187,7 @@ jobs:
       search_index: ${{ steps.filter.outputs.search_index == 'true' || env.mustTrigger == 'true' }}
       serverless: ${{ steps.filter.outputs.serverless == 'true' || env.mustTrigger == 'true' }}
       stream: ${{ steps.filter.outputs.stream == 'true' || env.mustTrigger == 'true' }}
+      control_plane_ip_addresses: ${{ steps.filter.outputs.control_plane_ip_addresses == 'true' || env.mustTrigger == 'true' }}
```

Comment:
are we also creating the resource for CFN and CDK?

Reply:
Given that it is only a get request (and therefor a single data source in tf) I believe this could not be added as a resource in CFN.